src/integrationtests/java/com/aws/greengrass/integrationtests/e2e/BaseE2ETestCase.java

```diff
@@ -226,7 +226,7 @@ protected BaseE2ETestCase() throws Exception {
     }
 
     public static void setDefaultRunWithUser(Kernel kernel) {
-        new DeviceConfiguration(kernel).getRunWithDefaultPosixUser().dflt("nobody");
+        new DeviceConfiguration(kernel).getRunWithDefaultPosixUser().dflt("faham");
```

Comment:
undo?

Reply:
my bad, reverted changes made to BaseE2ETestCase